test/distrib/bazel/cpp/greeter_client.cc

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
is this a verbatim copy of greeter_client.cc from examples? if so, we should at least add a TODO to deduplicate.
I don't think we want to have multiple copies of greeter_client.cc in the repo.

Same for greeter_server.cc and the .proto file.

Reply:
greeter_client.cc is the same, but greeter_server.cc had to be modified.

> I don't think we want to have multiple copies of greeter_client.cc in the repo.

I don't know if I agree with this. greeter_client and greeter_server are the two simplest usages of gRPC possible, by definition. I could turn greeter_client.cc into a symlink (I think) but I'm not sure how much value we would derive from that.

Reply:
I still think there's value in de-duplicating copies of greeter_*.cc, even though this is the "simplest possible" usage, it's still 100 LOC of not entirely trivial code.
But we can leave as is for now (I'd appreciate ideas for how to deduplicate in the future though, having a symlink doesn't seem that bad actually, but it would have to be for all files - client, server and the .proto file)

Reply:
Ah, I have to correct myself. All three of these files actually had to be modified. The proto lives at a different path from in the parent repo, so the `#includes` and proto package differ from what exists in the examples directory (as they should, since the package should mirror the directory structure).

The server also had to be changed to not use a fixed port. This would lead to port clashes and test failures when running concurrently. But you don't want to make the quality of the example code _worse_ by making changes for test considerations.